engine/runtime/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -548,12 +567,46 @@ class Compiler(
     uncachedParseModule(module, isGenDocs)
   }
 
+  private val ieb: scala.collection.mutable.Map[Module, BindingsMap] =
+    scala.collection.mutable.HashMap.empty;
+
   /** Retrieve module bindings from cache, if available.
     *
     * @param module module which is conssidered
-    * @return module's bindings, if available in libraries' bindings cache
+    * @return module's bindings, either from cache or from the IR
     */
-  def importExportBindings(module: Module): Option[BindingsMap] = {
+  def importExportBindings(module: Module): BindingsMap = {
+    var m = ieb.get(module)
```

Comment:
You likely want `getOrElseUpdate` instead

Reply:
Removed in https://github.com/enso-org/enso/commit/143e4f9a9ee632260ab7eb0ce0b8b887fe20aecb to narrow the scope of the PR down.